app/controllers/materials_controller.rb

```diff
@@ -1,8 +1,11 @@
+include Pagy::Backend
```

Comment:
This should be inside the class below otherwise it is polluting the global namespace in ruby.